_data/definitions.yml

```diff
@@ -0,0 +1,14 @@
+BOVS:
+  Node: A vertex in a BOVS diagram. Nodes function like nouns, representing Parties.
+  Arrow: An edge in a BOVS diagram. Arrows function like verbs, connecting Nodes to show relationships.
+  Chain: A series of Nodes continuously connected by Arrows is called a Chain. The "start" of the Chain is the first Node and the "end" is the last, based on the direction of the Arrows.
+  Party: General term for any kind of thing (e.g. Person, Company) that can be involved in a beneficial ownership situation.
+  Beneficiary: Any Party that has ownership or control of another Party. The Beneficiary at the start of a Chain is the "ultimate" Beneficiary.
+  Subject: Any type of Party that cannot be legally owned or controlled (e.g. a Person). Subjects are always the ultimate Beneficiaries, i.e always appear at the start of a Chain.
+  Object: Any type of Party that must be legally owned or controlled (e.g. a Company). Objects can appear at any point in a Chain except the start.
+  Person: A single private individual.
+  Regime: A country, state, or governing body with recognised sovereignty that can own or control Objects.
```

Comment:
I think we should be using the term **Jurisdiction** here, rather than 'Regime' as:

(a) It fits the definition better;

(b) We use the term 'Disclosure regime' elsewhere to mean 'Any system of disclosure rules and practices, whether set by a national government, a sectoral body, or some other actor' - so it would be good to avoid other uses of 'regime'